.env

```diff
@@ -16,4 +16,6 @@ REACT_APP_AUTH_URL=
 REACT_APP_CLIENT_ID=
 REACT_APP_AUTH_REALM=
 REACT_APP_PORTAIL_URL=
-REACT_APP_IDENTITY_PROVIDER=
\ No newline at end of file
+REACT_APP_IDENTITY_PROVIDER=
+# REACT_APP_NO_ACTIVITY_TIMEOUT: when oidc enabled, app logout user after REACT_APP_NO_ACTIVITY_TIMEOUT (number in seconds), default 900 seconds
+REACT_APP_NO_ACTIVITY_TIMEOUT=
```

Comment:
I think this should not be a props but reflect the refresh token expiration time.

Reply:
you're right